src/sentry/models/groupsubscription.py

```diff
@@ -12,7 +12,9 @@
 
 
 class GroupSubscriptionReason(object):
-    implicit = -1  # not for use as a persisted field value
+    committed = -2  # not for use as a persisted field value
+    implicit = -1   # not for use as a persisted field value
```

Comment:
This feels like the wrong abstraction point for these now, I might have a follow up to move them to a better location later.